vehicle/mkv/software/air_control/tests/test_initial_checks.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import time
+from formula.projects.microhitl import Values as PinValue
+
+
+def reset(iocontroller, pins):
+    iocontroller.write_pin(pins["RESET"][0], PinValue.LOW)
+    time.sleep(0.5)
+    iocontroller.write_pin(pins["RESET"][0], PinValue.HIGH)
+
+
+def test_bms_can_timeout(canbus, iocontroller, pins):
+    """
+    Fault during init if BMS message takes longer than 1 second to receive
+    """
+    reset(iocontroller, pins)
+    assert canbus.get_state("air_fault") == "NONE"
+    time.sleep(1.1)
+    assert canbus.get_state("air_state") == "FAULT"
+    assert canbus.get_state("air_fault") == "CAN_BMS_TIMEOUT"
+
+
+def test_bms_voltage_low(canbus, iocontroller, pins):
+    """
+    Fault during init if BMS voltage is less than 200V
+    """
+    canbus.set_state("pack_voltage", 100)
+    canbus.set_periodic("bms_core", 0.3)
+
+    reset(iocontroller, pins)
```

Comment:
There is a fair amount of similar code among this and the other test functions in this file, presumably because you're checking for various faults during init. Consider making a common instance of the canbus in a setup function with sane/reasonable defaults, and then just explicitly set the values that are relevant to each test.